HRIS.Services.Tests/Helpers/EmailHelperUnitTests.cs

```diff
@@ -19,7 +19,7 @@ public EmailHelperUnitTests()
         _helper = new EmailHelper(_db.Object);
     }
 
-    [Fact]
+    [Fact(Skip="Needs to be fixed/updated")]
```

Comment:
```suggestion
    [Fact]
```
![image](https://github.com/RetroRabbit/RGO-Server/assets/123597519/2d540936-94bf-48a0-997e-ad59e08dfd25)